template-sample-app-3/template/src/components/layers/TilesetLayer.js

```diff
@@ -58,6 +58,7 @@ function TilesetLayer() {
             layerAttributes: { ...layerConfig },
           })
         );
+        cartoLayerProps?.onDataLoad && cartoLayerProps.onDataLoad(data);
```

Comment:
👍🏻

Reply:
Maybe change it to `cartoLayerProps.onDataLoad && cartoLayerProps.onDataLoad(data);`? I think `cartoLayerProps` cannot be null/undefined

Reply:
agree, applied suggested change